pinkopy/base_session.py

```diff
@@ -137,7 +137,7 @@ def request(self, method, path, attempt=None, headers=None, payload=None,
             else:
                 log.info('CMDBSession made request to {0}'.format(url))
                 return res
-        except requests.exceptions.HTTPError as err:
+        except requests.HTTPError as err:
```

Comment:
I don't see the intent of changing to remove `exceptions`. It seems more explicit to me, but since `requests.HTTPError` is an alias to `requests.exceptions.HTTPError`, I don't care. If it makes you happy, I'm happy. :smile: